src/Components/CreateImageWizard/formComponents/ImageOutputReleaseSelect.js

```diff
@@ -15,6 +15,7 @@ const ImageOutputReleaseSelect = ({ label, isRequired, ...props }) => {
   const { change, getState } = useFormApi();
   const { input } = useFieldApi(props);
   const [isOpen, setIsOpen] = useState(false);
+  const [isExpanded, setIsExpanded] = useState(false);
```

Comment:
isExpanded is a little too generic/vague imo, how about showCentos/setShowCentos ? Or showDevelopmentOptions/...?

Reply:
Agreed 100%! Went for showDevelopmentOptions in case we add Fedora in the future.